ios/Podfile.lock

```diff
@@ -1055,8 +1055,6 @@ PODS:
     - Firebase/Core (= 6.14.0)
     - Firebase/MLVision (= 6.14.0)
     - Firebase/MLVisionFaceModel (= 6.14.0)
-    - FirebaseMLCommon (= 0.19.0)
```

Comment:
It will cause FaceDetector to stop working. Is that the intention?

Reply:
No it doesn't, why do you think that? `Firebase/MLVision` implicitly imports `FirebaseMLCommon`

Reply:
If so, that's ok. Thanks!